diffus/src/edit/collection.rs

```diff
@@ -1,5 +1,7 @@
 use crate::Same;
 
+// FIXME references ?
+// FIXME T::D ?
```

Comment:
Can probably remove these

Reply:
yes, there already is issues discussing this, and the reference thing was settled :)